src/data/faq_de.json

```diff
@@ -146,6 +146,23 @@
                             "Unter dem Reiter Zertifikate kann die App nur '<b>EU Digitale</b> COVID-Zertifikate' scannen.",
                             "Siehe auch folgenden Eintrag zu <a href='#vac_cert_invalid' target='_self' rel='noopener'>ungültigen Impfzertifikaten</a>."
                          ]
+                    },
+                    {
+                        "title": "Wie lange ist mein Zertifikat gültig?​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.&nbsp;h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand von Juli 2021.",
+                            "Wir unterscheiden grundsätzlich​<ul><li>die Gültigkeit des digitalen Zertifikats (die <b>technische Gültigkeit</b>) und</li><li>die Gültigkeit des Impfschutzes (die <b>fachliche Gültigkeit</b>) im Sinne der 'Dauer des Schutzes'</li></ul>",
+                            "<b>Technische Gültigkeit</b>",
+                            "Das Zertifikat als Dokument hat ein Ausstellungsdatum und ein technisches Ablaufdatum, so wie auch Ihr Reisepass oder Personalausweis beispielsweise. Für alle Zertifikate gilt, dass das Ablaufdatum vom Herausgeber des Zertifikats gesetzt wird. Derzeit werden in Deutschland die digitalen Impf-, Test- und Genesenen-Zertifikate der EU in der Regel mit einer technischen Gültigkeit von einem Jahr ab Ausstellungsdatum ausgestellt.​",
+                            "<b>Fachliche Gültigkeit</b>",
+                            "In dem Zertifikat werden Daten transportiert, z.&nbsp;B. in dem Fall des Impfzertifikats, Informationen zur Impfung. Diese Impfung fand an einem Datum X statt und gilt ab dann, nach aktuellem Stand der wissenschaftlichen Erkenntnisse, z.&nbsp;B. in Deutschland ein Jahr. Danach gilt der Impfschutz als erloschen. Die Dauer des Impfschutzes bezeichnen wir als 'fachliche Gültigkeit'. Die fachliche Gültigkeit der Impfung zählt also ab dem letzten Impfdatum bzw. genau genommen, ab dem Tag nach Erreichen der Wirksamkeit des Impfschutzes (14 Tage nach Datum X).​ Während in Deutschland derzeit die fachliche Gültigkeit auf ein Jahr festgelegt ist, kann die Gültigkeit des Impfschutzes in anderen Ländern davon abweichen.​",

```

Comment:
"14 Tage nach Datum X"
It is actually 15 days, not 14 days.
14 full days have to have expired between the day of the vaccination and the day that the vaccination is declared to be complete.

See https://digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen


>Warum ist der QR-Code als Nachweis für den vollständigen Impfschutz erst nach zwei Wochen gültig?

In Deutschland gelten Personen als geschützt, die mit einem in der EU zugelassenen COVID-19-Impfstoff geimpft wurden und bei denen nach Gabe der letzten Impfstoffdosis mindestens 14 Tage vergangen sind (Ausnahmen bzgl. Genesener siehe https://www.rki.de/SharedDocs/FAQ/COVID-Impfen/gesamt.html).

>Wenn Sie zum Beispiel die letzte notwendige Corona-Impfung am 01. Juni erhalten haben, dann ist der vollständige Corona-Impfschutz erst am 16. Juni erreicht.

Reply:
right. thanks